package.json

```diff
@@ -43,10 +43,12 @@
     "nodemon": "^1.11.0",
     "aurelia-tools": "^0.2.4",
     "babel-eslint": "^7.1.1",
+    "debounce": "^1.0.2",
```

Comment:
do we need these dependencies here or is it sufficient that the CLI adds these deps to the package.json that it creates during `au new`?